zstacklib/zstacklib/utils/sanlock.py

```
@@ -533,6 +533,28 @@ def get_lockspace(vg_uuid):
     return None
 
 
+def get_lockspace_sector_size(vg_uuid):
+    dev_path = "/dev/mapper/%s-lvmlock" % vg_uuid
+    try:
+        physical_block_size = linux.get_dev_physical_sector_size(dev_path)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)